Client/src/Pages/Incidents/index.jsx

```diff
@@ -1,9 +1,72 @@
-import React from 'react'
+import { styled } from '@mui/material/styles';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import StatusLabel from "../../Components/Label/";
```

Comment:
Currently doesn't compile as `Label` has no default export, so unless we provide one we need to import it as a named import:

import { StatusLabel } from "../../Components/Label/";